lib/pbench/cli/server/repair.py

```diff
@@ -284,7 +313,7 @@ def repair_metadata(kwargs):
     "--progress", "-p", type=float, default=0.0, help="Show periodic progress messages"
 )
 @click.option(
-    "--verify", "-v", default=False, is_flag=True, help="Display intermediate messages"
+    "--verify", "-v", default=False, count=True, help="Display intermediate messages"
```

Comment:
`count` is a boolean `click.Option` parameter that causes click to count occurrences of the option and resolve to the count rather than simply presence. In short, the traditional `-vv` behavior. I considered making `-v` take an optional integer instead; a bit less weird if also less traditional.

Reply:
Ah!  `count` is an an option/parameter to `option()`.  Nevermind.  🙂

I'm accustomed to `-v` taking a value in this instance; knowing that it's supposed to count the `-v`'s works.